packages/angular/src/utils/mf/utils.ts

```diff
@@ -48,7 +48,14 @@ export function getFunctionDeterminateRemoteUrl(isServer: boolean = false) {
   const remoteEntry = isServer ? 'server/remoteEntry.js' : 'remoteEntry.mjs';
 
   return function (remote: string) {
-    const remoteConfiguration = readCachedProjectConfiguration(remote);
+    let remoteConfiguration = null;
+    try {
+      remoteConfiguration = readCachedProjectConfiguration(remote);
+    } catch (e) {
+      throw new Error(
+        `Cannot find remote "${remote}". Check that the remote name is correct in your module-federation.config.js.\n`
```

Comment:
It could be `module-configuration.ts`, probably just say `module federation config file`.